tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/wizard/NewLiferayModuleProjectWizard.sdef

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
All module wizard stuff should go in a new package. "com.liferay.ide.project.ui.modules"